prism_compile.c

```diff
@@ -3387,6 +3387,9 @@ pm_compile_builtin_attr(rb_iseq_t *iseq, const pm_scope_node_t *scope_node, cons
         else if (strcmp(RSTRING_PTR(string), "use_block") == 0) {
             iseq_set_use_block(iseq);
         }
+        else if (strcmp(RSTRING_PTR(string), "c_trace") == 0) {
+            ISEQ_BODY(iseq)->builtin_attrs |= BUILTIN_ATTR_C_TRACE;
+        }
```

Comment:
What does this do? Can you add a short comment?

Reply:
I thought documenting it above the `BUILTIN_C_TRACE` definition was enough, but I copied it here for you. https://github.com/ruby/ruby/pull/11955/commits/a378fb0727781a6130aed5a3cadc856c66e578f4